### PR TITLE
Updated error messages for invalid phone pin and update timeout

### PR DESCRIPTION
**Pull request by bransteitterbr**
Updated per comments provided in https://github.com/episphere/connect/issues/682

### CODE REVIEW

**Review by JoeArmani — APPROVED** [review]
looks good

File: js/settingsHelpers.js [hunk 1]
```diff
@@ -533,7 +533,7 @@ export const addOrUpdateAuthenticationMethod = async (email, phone, userData) =>
     try {
       await updateFirebaseAuthEmail(email);
     } catch (error) {
-      console.error(`Error: updateFirebaseAuthEmail(): ${error}`);
+      console.error(`Error: updateFirebaseAuthEmail(): ${error}`, error);
```
Comment (we-ai) [inline js/settingsHelpers.js:536]:
```suggestion
      console.error(`Error: updateFirebaseAuthEmail().`, error);
```

**Review by we-ai — CHANGES_REQUESTED** [review]
Looks good. Minor change suggested.

File: js/settingsHelpers.js (hunk 1 above)
Comment (bransteitterbr) [reply js/settingsHelpers.js]: Updated

**Review by we-ai — APPROVED** [review]

**Pull request merged**